packages/web/src/components/gcds-link/stories/overview.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Link from './gcds-link.stories';
+
+<Meta of={Link} name="Overview" />
+
+# Link<br/>`<gcds-link>`
+
+_Also called: call to action, CTA, link._
+
+A Link is an interactive object that highlights an important or common action.
+
+<Canvas
+  of={Link.Default}
+  story={{ inline: true }}
+/>
+
+## Examples
+<br/>
+
+### External
+<Canvas
+  of={Link.External}
+  story={{ inline: true }}
+/>
+
+### Download
+<Canvas
+  of={Link.Download}
+  story={{ inline: true }}
+/>
+
+### Email addresses
+<Canvas
+  of={Link.Email}
+  story={{ inline: true }}
+/>
+
+### Phone numbers
+<Canvas
+  of={Link.Phone}
+  story={{ inline: true }}
+/>
+
+### Size
+<Canvas
+  of={Link.Sizes}
+  story={{ inline: true }}
+/>
```

Comment:
Could you add another example here of a regular-sized link, please? Showing both options makes it a bit easier to see the difference.